### DISCUSSION TITLE
Поиск стратегии через веб

### DISCUSSION THREAD

**Discussion by bear-gha** [Anonym-tsk/nfqws-keenetic#122] [Идеи и предложения]
Как одна из возможных идей на будущее - можно сделать поиск стратегии через веб интерфейс, все равно на основном пакете оно не завязано, который будет стопиться. А конфиг уже там можно менять, поэтому эта часть уже не требуется.

**Comment by deadcxap** [comment]
Тогда бы и конструктор конфига в виде галочек, как в полноценном гуе)

**Comment by unameras** [comment]
А ещё было бы неплохо конструктор стратегий или фильтр, чтобы можно было из кучи найденных стратегий оставить одну, которая подходит ко всем сайтам из поиска. Грубо говоря поиск дубликатов и вывод той стратегии, которая подходит.
Пример: 
сайты A, B и C.
найденные стратегии:
A: 1, 2, 3, 4, 5
B: 3, 5, 6, 7
C: 3, 4, 6, 7, 8, 9. 
Общая для всех только 3, значит её нужно выделить/подсветить/вывести отдельно.
